### ISSUE TITLE
Unfollow not working good 

### ISSUE THREAD

**Issue by yonimm** [instabot-py/instabot.py#374]
I used mode 0 and i have more than 1500 following 
Mod 5 not working for me
How i can unfollow all the users?

**Comment by ekremgurdal** [comment]
I think there is a bug in mode 5. Can you try to change like this? ->

**Comment by yonimm** [comment]
not work for me.
any solution? i have more then 2k :)

**Comment by ekremgurdal** [comment]
Can you specify your issue? Can you upload a screenshot?

**Comment by mactrash** [comment]
setup as Follow 300, unfollow 1000 ... result 1000+ unfollower remain after running few days

**Comment by dadealer** [comment]
same problem at my acc

**Comment by mactrash** [comment]
@yonimm @dadealer use some unfollow apps to do the unfollow otherwise it will keep growing

**Comment by dadealer** [comment]
@mactrash but it is not the solution

**Comment by loganyc1934** [comment]
My unfollow functionality gets blocked by Instagram if I do `limit = random.randint(10,999999)+1`. What's this `limit`? Don't quite understand the code for mode 5 and `unfollow_protocol`.

**Comment by ugurozturk** [comment]
Major changes have been made to Instabot.py. If this is still an issue, please open a new ticket.
